src/cr/cube/cube.py

```diff
@@ -645,6 +717,9 @@ class _UnweightedCountMeasure(_BaseMeasure):
     @lazyproperty
     def _flat_values(self):
         """tuple of int counts before weighting."""
+        # ---If valid_count are expressed in the cube dict, returns its data---
+        if self._cube_dict["result"]["measures"].get("valid_count", {}).get("data"):
+            return tuple(self._cube_dict["result"]["measures"]["valid_count"]["data"])
```

Comment:
This is a little confusing. Our canonical way of naming things is that "weighted" is implicit, and that "unweighted" is explicit. So I'd rather change the keys altogether, and call the weighted version of the valid counts just `"valid_count"`, and the unweighted `"valid_count_unweighted"`, so that we don't confuse ourselves.

And then here, in this property, you can use the "unweighted" explicitly, which will correspond with the class name. And in the other one (maybe) you'd use just valid counts.